docs/migration-to-NRI.md

```diff
@@ -0,0 +1,113 @@
+  - ```console
+    systemctl restart containerd
+  - ```console
+    systemctl restart kubelet
+    ```
+
+- ### For CRI-O the process is the same, but change it to the crio socket and then restart the crio service.
+
+- ### For containerd edit the containerd config file and look for the section "io.containerd.nri.v1.nri" and replace "disable = true" with "disable = false":
+  - ```console
+    vim /etc/containerd/config.toml
+    ```
+
+- ### For crio edit the crio config and look for the "[crio.nri]" section, replace "enable_nri = false" with "enable_nri = true" and uncomment the NRI config lines.
+  - ```console
+    sudo vim /etc/crio/crio.conf
+    ```
+
+- ### Build the policies:
+  - ```console
+    git clone https://github.com/jukkar/nri-resmgr.git
+  - ```console
+    cd nri-resmgr
+    ````
+  - Switch to a stable branch
+  - ```console
+    git checkout preview
+  - ```console
+    make
+  - ```console
+    make images
+    ````
+
+- ### Apply required crds:
+  - ```console
+    kubectl apply -f deployment/base/crds/noderesourcetopology_crd.yaml
+    ````
+
+- ### Import the image and start the NRI plugin you want to run:
+  - For Containerd:
+  - ```console
+    ctr -n k8s.io images import build/images/nri-resmgr-topology-aware-image-9797e8de7107.tar
+  - For CRI-O:
+  - <b>See step 3 how to push the images to a registry, then pull from there.</b>
+  - Then deploy the plugin:
+  - ```console
+    kubectl apply -f build/images/nri-resmgr-topology-aware-deployment.yaml
+    ```
+
+- ### Deploy your test pod.
+
+- ### See the resources the pod got assigned with:
+  - ```console
+    kubectl exec $pod -c $container  -- grep allowed_list: /proc/self/status
+  - ```console
+    Output should look similar to the output of CRI-RM
+    ```
+
+## 3. Steps for a more real-life migration using self-hosted image repository.
+- ### Same steps as above for enabling NRI with Containerd/CRI-O and building the images.
+- ### Load the .tar file into your image repository:
+  - ```console
+    docker load -i build/images/nri-resmgr-topology-aware-image-9797e8de7107.tar
+    ````
+
+- ### Tag the image with your registry:
+  - ```console
+    docker tag myimage:<version> myregistry.com/myimage:<version>
+    ```
+
+- ### Push the image to your registry:
+  - ```console
+    docker push myregistry.com/myimage:<version>
+    ```
+
+- ### Remember to change the image name & pull policy in the plugins .yaml file to match your registyr and image, ex:
+  - ```console
+    vim build/images/nri-resmgr-topology-aware-deployment.yaml
+    ```
+
+- ### Then deploy the plugin simlarly to the earlier step.
+
+## 4. Migrating existing configuration
+- The ConfigMap used by the ported policies/infra has a different name/naming scheme than the original one used in CRI-RM, ex:
+  - configMapName: "configmap-name: cri-resmgr-config" is now "configmap-name: nri-resmgr-config"
```

Comment:
I think we should mention here that the details of grouping nodes by labeling to share configuration is also slightly different. What used to be
     `cri-resource-manager.intel.com/group: $GROUP_NAME`
is now
    `nri-resmgr.intel.com/group: $GROUP_NAME`